lib/samples/show_device_location_history/show_device_location_history_sample.dart

```diff
@@ -160,19 +180,44 @@ class _ShowDeviceLocationHistorySampleState
     });
 
     try {
+      // Start the location data source.
       await _locationDataSource.start();
-      _locationSubscription = _locationDataSource.onLocationChanged
-          .listen(_handleLdsLocationChange);
+      // Listen for location changes.
+      _locationSubscription = _locationDataSource.onLocationChanged.listen(
+        _handleLdsLocationChange,
+      );
```

Comment:
The call to `onLocationChanged.listen` should move after the try/catch block. It can't throw an ArcGISException, so there's no reason for it to be a part of the try/catch block.